hscontrol/templates/apple.go

```diff
@@ -0,0 +1,240 @@
+package templates
+
+import (
+	"fmt"
+
+	"github.com/chasefleming/elem-go"
+	"github.com/chasefleming/elem-go/attrs"
+)
+
+func HeadscaleApple(url string) *elem.Element {
+	return elem.Html(nil,
+		elem.Head(attrs.Props{
+			attrs.Lang: "en",
+		},
+			elem.Meta(attrs.Props{
+				attrs.Charset: "UTF-8",
+			}),
+			elem.Meta(attrs.Props{
+				attrs.HTTPequiv: "X-UA-Compatible",
+				attrs.Content:   "IE=edge",
+			}),
+			elem.Meta(attrs.Props{
+				attrs.Name:    "viewport",
+				attrs.Content: "width=device-width, initial-scale=1.0",
+			}),
+			elem.Title(nil,
+				elem.Text("headscale - Apple")),
+			elem.Style(nil,
+				elem.CSS(`
+      body {
+        margin: 40px auto;
+        max-width: 800px;
+        line-height: 1.5;
+        font-size: 16px;
+        color: #444;
+        padding: 0 10px;
+        font-family: Sans-serif;
+      }
+
+      h1,
+      h2,
+      h3 {
+        line-height: 1.2;
+      }
+          `),
+			),
+		),
+		elem.Body(nil,
+			elem.H1(nil,
+				elem.Text("headscale: macOS configuration")),
+			elem.H2(nil,
+				elem.Text("Recent Tailscale versions (1.34.0 and higher)"),
+			),
+			elem.P(nil,
+				elem.Text("Tailscale added Fast User Switching in version 1.34 and you can now use the new login command to connect to one or more headscale (and Tailscale) servers. The previously used profiles does not have an effect anymore."),
+			),
+			elem.H3(nil,
+				elem.Text("Command line"),
+			),
+			elem.P(nil,
+				elem.Text("Use Tailscale's login command to add your profile:"),
+			),
+			elem.Pre(nil,
+				elem.Code(nil,
+					elem.Text(
+						fmt.Sprintf("tailscale login --login-server %s", url)),
+				),
+			),
+			elem.H3(nil,
+				elem.Text("GUI"),
+			),
+			elem.Ol(nil,
+				elem.Li(nil,
+					elem.Text("ALT + Click the Tailscale icon in the menu and hover over the Debug menu")),
+				elem.Li(nil,
+					elem.Text("Under \"Custom Login Server\", select \"Add Account...\"")),
```

Comment:
When having a string in go that uses `"`, you should do:

```suggestion
					elem.Text(`Under "Custom Login Server", select "Add Account..."`)),
```

to not have to escape everything.